accessibility-checker-engine/help/HAAC_Video_HasNoTrack.mdx

```diff
@@ -21,13 +21,15 @@ Media embedded with a `<video>` element must have a `<track>` element to provide
 
 ### Why is this important?
 
-Text tracks contain a textual transcription of either audio (captions) or visual (visual/video description) aspects of video or audio content in a synchronized media. This may include the dialog, sound effects, and relevant musical cues, or information about characters, location, actions and scene changes. Providing text alternatives makes information available that some people would not have access to otherwise.
+Text tracks or open captions contain a textual transcription of either audio (captions) or visual (visual/video description) aspects of video or audio content in a synchronized media. This may include the dialog, sound effects, and relevant musical cues, or information about characters, location, actions and scene changes. Providing text alternatives makes information available that some people would not have access to otherwise.
 
 <div id="locSnippet"></div>
 
 ### What to do
 
-Use the `<track>` element with the `kind` attribute to provide text tracks for the audio (`kind="captions"`) and visual (`kind="descriptions"`) aspects of the video
+* Use the `<track>` element with the `kind` attribute to provide text tracks for the audio (`kind="captions"`) and visual (`kind="descriptions"`) aspects of the video;
+* OR make sure the video has no audio;
```

Comment:
What about the visual aspects?  Aren't they still needed?  Should we say: 

OR, if the video has no audio then visual description is needed but captions are not.

Reply:
@sharitrewin do you mean the audio descriptions of the video? that is covered by 1.2.3 and 1.2.4 for which we have a different rule - media-possible-video-described, do you think we should mention it here as well?

Reply:
ok i see what you mean, have made the changes